flutter/example/lib/main.dart

```diff
@@ -231,26 +230,6 @@ class CocoaExample extends StatelessWidget {
   }
 }
 
-class WebExample extends StatelessWidget {
-  const WebExample({Key? key}) : super(key: key);
-
-  final channel = const MethodChannel('example.flutter.sentry.io');
-
-  @override
-  Widget build(BuildContext context) {
-    return Column(
-      children: [
-        RaisedButton(
-          child: const Text('Web: console.log'),
-          onPressed: () async {
-            await channel.invokeMethod<void>('console.log', 'log me');
```

Comment:
I think we don't need this. It just called `Sentry.captureMessage`.

Reply:
For Web we also don't need method channels, just platform dependent imports.

Reply:
mm makes sense, but what's about if I have a JS file that does something for me, not written in Dart? would that be done thru method channels, right?

Reply:
Nope, as far as I know you can use https://api.dart.dev/stable/2.12.2/dart-js/dart-js-library.html for that. On web there is no method channel because Dart gets transpiled to JS.

Reply:
I guess we could offer support for that in Sentry even without Flutter.

Reply:
yep, thanks